# Covariance matrix completion via auxiliary information

Published: 2024-02-08T15:52:27Z

[Link to the paper](http://arxiv.org/abs/2402.05767v1)

## Authors

- Joseph Steneman
- Giuseppe Vinci


## Summary
  Covariance matrix estimation is an important task in the analysis of
multivariate data in disparate scientific fields, including neuroscience,
genomics, and astronomy. However, modern scientific data are often incomplete
due to factors beyond the control of researchers, and data missingness may
prohibit the use of traditional covariance estimation methods. Some existing
methods address this problem by completing the data matrix, or by filling the
missing entries of an incomplete sample covariance matrix by assuming a
low-rank structure. We propose a novel approach that exploits auxiliary
variables to complete covariance matrix estimates. An example of auxiliary
variable is the distance between neurons, which is usually inversely related to
the strength of neuronal correlation. Our method extracts auxiliary information
via regression, and involves a single tuning parameter that can be selected
empirically. We compare our method with other matrix completion approaches via
simulations, and apply it to the analysis of large-scale neuroscience data.



# Fast Signal Region Detection with Application to Whole Genome Association Studies

Published: 2023-05-14T14:45:39Z
Updated: 2024-02-08T14:11:59Z

[Link to the paper](http://arxiv.org/abs/2305.08172v2)

## Authors

- Wei Zhang
- Fan Wang
- Fang Yao


## Summary
  Research on the localization of the genetic basis associated with diseases or
traits has been widely conducted in the last a few decades. Scan methods have
been developed for region-based analysis in whole-genome association studies,
helping us better understand how genetics influences human diseases or traits,
especially when the aggregated effects of multiple causal variants are present.
In this paper, we propose a fast and effective algorithm coupling with
high-dimensional test for simultaneously detecting multiple signal regions,
which is distinct from existing methods using scan or knockoff statistics. The
idea is to conduct binary splitting with re-search and arrangement based on a
sequence of dynamic critical values to increase detection accuracy and reduce
computation. Theoretical and empirical studies demonstrate that our approach
enjoys favorable theoretical guarantees with fewer restrictions and exhibits
superior numerical performance with faster computation. Utilizing the UK
Biobank data to identify the genetic regions related to breast cancer, we
confirm previous findings and meanwhile, identify a number of new regions which
suggest strong association with risk of breast cancer and deserve further
investigation.



# Enhancing Cardiovascular Disease Risk Prediction with Machine Learning Models

Published: 2024-01-29T19:08:33Z
Updated: 2024-02-08T10:42:53Z

[Link to the paper](http://arxiv.org/abs/2401.17328v2)

## Authors

- Farnoush Shishehbori
- Zainab Awan


## Summary
  Cardiovascular disease remains a leading global cause of mortality,
necessitating accurate risk prediction tools. Traditional methods, such as
QRISK and the Framingham heart score, exhibit limitations in their ability to
incorporate comprehensive patient data, potentially resulting in incomplete
risk factor consideration. To address these shortcomings, this study explores
the application of machine learning models. Machine learning models, such as
support vector machines, and Random Forest, as well as deep learning techniques
like convolutional neural networks and recurrent neural networks, have emerged
as promising alternatives. These models offer superior performance,
accommodating a broader spectrum of variables and providing precise
subgroup-specific predictions. While machine learning integration holds promise
for enhancing risk assessment, it presents challenges such as data requirements
and computational constraints. Additionally, large language models have
revolutionised healthcare applications, augmenting diagnostic precision and
patient care. This study examines the core aspects of cardiovascular disease
event risk and presents a thorough review of traditional and machine learning
models, alongside deep learning techniques, for improved accuracy. It offers a
comprehensive survey of relevant datasets, critically compares ML models with
conventional approaches, and synthesizes key findings and themes, highlighting
their implications for clinical practice. Furthermore, the potential of machine
learning and large language models in cardiovascular medicine is undeniable.
However, rigorous validation and optimisation are imperative before widespread
application in healthcare. This integration promises more accurate and
personalised cardiovascular care.



# Machine learning applied to omics data

Published: 2024-02-08T10:22:45Z

[Link to the paper](http://arxiv.org/abs/2402.05543v1)

## Authors

- Aida Calvi√±o
- Almudena Moreno-Ribera
- Silvia Pineda


## Summary
  In this chapter we illustrate the use of some Machine Learning techniques in
the context of omics data. More precisely, we review and evaluate the use of
Random Forest and Penalized Multinomial Logistic Regression for integrative
analysis of genomics and immunomics in pancreatic cancer. Furthermore, we
propose the use of association rules with predictive purposes to overcome the
low predictive power of the previously mentioned models. Finally, we apply the
reviewed methods to a real data set from TCGA made of 107 tumoral pancreatic
samples and 117,486 germline SNPs, showing the good performance of the proposed
methods to predict the immunological infiltration in pancreatic cancer.



# Assumption-lean and Data-adaptive Post-Prediction Inference

Published: 2023-11-23T22:41:30Z
Updated: 2024-02-06T21:23:09Z

[Link to the paper](http://arxiv.org/abs/2311.14220v3)

## Authors

- Jiacheng Miao
- Xinran Miao
- Yixuan Wu
- Jiwei Zhao
- Qiongshi Lu


## Summary
  A primary challenge facing modern scientific research is the limited
availability of gold-standard data which can be both costly and labor-intensive
to obtain. With the rapid development of machine learning (ML), scientists have
relied on ML algorithms to predict these gold-standard outcomes with easily
obtained covariates. However, these predicted outcomes are often used directly
in subsequent statistical analyses, ignoring imprecision and heterogeneity
introduced by the prediction procedure. This will likely result in false
positive findings and invalid scientific conclusions. In this work, we
introduce an assumption-lean and data-adaptive Post-Prediction Inference
(POP-Inf) procedure that allows valid and powerful inference based on
ML-predicted outcomes. Its "assumption-lean" property guarantees reliable
statistical inference without assumptions on the ML-prediction, for a wide
range of statistical quantities. Its "data-adaptive'" feature guarantees an
efficiency gain over existing post-prediction inference methods, regardless of
the accuracy of ML-prediction. We demonstrate the superiority and applicability
of our method through simulations and large-scale genomic data.



